website/client/src/components/members/memberSearchDropdown.vue

```diff
@@ -61,6 +61,12 @@ export default {
     selectMember (member) {
       this.$emit('member-selected', member);
     },
+    memberName (member) {
+      if (member.auth.local.username) {
```

Comment:
I think the check here should be `member.auth.local && member.auth.local.username` because old users that signed up using facebook or google and without a set username might have an empty `auth.local` object

Reply:
Thanks for the heads up, updated according to your suggestion!